pkg/kubelet/server/stats/summary.go

```diff
@@ -81,6 +81,31 @@ func (sp *summaryProviderImpl) Get(updateStats bool) (*statsapi.Summary, error)
 		Rlimit:           rlimit,
 		SystemContainers: sp.GetSystemContainersStats(nodeConfig, podStats, updateStats),
 	}
+
+	if onlyCPUAndMemory {
```

Comment:
Could you create a new function instead? I think it'd be easier to test and maintain down the road.

Reply:
done

Reply:
Uh...sorry for not being clear. I was thinking that we could add a `GetCPUMemoryStats` function, as opposed to adding another flag in the `Get` function.

Reply:
done